shared/components/Animation/__tests__/Reveal.spec.jsx

```diff
@@ -1,26 +1,37 @@
 import React from 'react'
-import { render } from 'enzyme'
+import { mount } from 'enzyme'
 
 import Reveal from '../Reveal'
 
 describe('Reveal', () => {
-  it('renders when the transition is triggered', () => {
-    const reveal = render(
-      <Reveal timeout={500} in={true} height={300}>
+  const doMount = (overrides = {}) =>
+    mount(
+      <Reveal {...overrides} timeout={500} height={300}>
         {() => <div>Content to reveal</div>}
       </Reveal>
     )
 
+  it('renders when the transition is triggered', () => {
+    const reveal = doMount({ in: true })
+
```

Comment:
Actually, I think we are better off doing a `render` for the snapshot instead of a `mount`. If you look at the snapshot now it is way large and contains the entire component tree, which is less useful for a snapshot like this. Having the entire tree makes it harder to see what's going on.

I recommend just leaving the snapshot tests the way they were before, as we care about all the props for the snapshot.